tests/test_tokenization_common.py

```diff
@@ -368,11 +369,21 @@ def test_sentencepiece_tokenize_and_convert_tokens_to_string(self):
         # check if converting back to original text works
         reverse_text = tokenizer.convert_tokens_to_string(tokens)
 
-        if self.test_sentencepiece_ignore_case:
-            reverse_text = reverse_text.lower()
+        # All tokenizers pass this test without the below commented out code.
+        # if self.test_sentencepiece_ignore_case:
+        #     reverse_text = reverse_text.lower()
```

Comment:
I just tested it locally, and I have no errors when uncommenting these lines. Do you specifically remember the error that caused you to comment out these lines?

Reply:
Oh I understand! I think it's because currently no tokenizer has the lowercasing argument enabled. I'd be in favor of leaving it as is in case a future tokenizer uses this setup

Reply:
No problem, I reverted the change.